elliottlib/cli/validate_advisory.py

```diff
@@ -0,0 +1,44 @@
+from elliottlib import brew, constants, errata, Runtime
+from elliottlib.cli.common import cli
+
+import click
+import json
+import koji
+import requests
+import textwrap
+pass_runtime = click.make_pass_decorator(Runtime)
+
+
+@cli.command("validate-advisory", short_help="Validate RHSA with Product Security checks")
+@click.argument('advisory', type=int)
+@pass_runtime
+def validate_advisory_cli(runtime, advisory):
+    """
+    Validates an RHSA by returning the 'alerts' generated by Product Security's central SFM2 server.
+
+    Usage:
+\b
+    $ elliott validate-advisory ID
+"""
+    session = requests.Session()
+    url = constants.SFM2_ERRATA_ALERTS_URL.format(id=advisory)
+    print(url)
+    print("Validating..")
+    print()
+    resp = session.get(url)
+
+    # Will fail if id invalid, not an RHSA or RHSA has embargoed data
+    if resp.status_code != 200:
+        print(f"HTTP {resp.status_code}: Could not validate advisory {advisory} (is it an RHSA? without embargo?)")
```

Comment:
Is there a period of time where the server is acquiring the data, and would a retry make sense here?
Would it make sense to query erratatool to disambiguate between the two (maybe three) reasons for status to be != 200?

Reply:
When the server receives a request, it will reload the advisory from ET before responding. So the returned Alerts should always be up-to-date.

I guess we could add an additional query to ET on error, but I'm not sure if that additional query/response time is worth the dis-ambiguity of errors.